operator/operator.go

```diff
@@ -95,6 +95,7 @@ func Setup(cfg *Config) *Operator {
 		PhaseDuration:    cfg.PhaseDuration,
 		ResponseDuration: cfg.ResponseDuration,
 		PowDepth:         cfg.PowDepth,
+		InitBlock:        1,
```

Comment:
You don't need that, `DeployContracts` should set the `InitBlock`. Also, you cannot know the correct init block number here.

Reply:
I will remove it.